### PR TITLE
Move issues to Github forms

### PR DESCRIPTION
**Pull request by McGiverGim**
Same change that in the firmware project: betaflight/betaflight#11322

GitHub now has a new feature to use forms in the issues tab.

The users have a lot of problems to fill the template in a correct way. Now it will be a lot easier.

This is the result:

image

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
